docs/database/db_schema-main.sql

```diff
@@ -59,6 +58,21 @@ CREATE TABLE `certificateStatus` (
   PRIMARY KEY (`serial`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `challenges` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `authorizationID` char(43) NOT NULL,
+  `LockCol` bigint(20) DEFAULT NULL,
+  `type` varchar(255) NOT NULL,
+  `status` varchar(255) DEFAULT NULL,
+  `error` varchar(255) DEFAULT NULL,
+  `validated` datetime DEFAULT NULL,
+  `uri` varchar(255) DEFAULT NULL,
+  `token` char(43) NOT NULL,
+  `tls` tinyint(1) DEFAULT NULL,
+  `validation` mediumblob,
```

Comment:
So I have made this consistent, but in the other way to what I think you meant now, I made all the auth ID fields `char(43)`.

With strict mode I think this makes sense, and stops us accidentally putting stuff that will get truncated into columns. The schemas can be updated by hand, it's not that bad (:/) and we'll have to be more rigorous about it anyway once gorp `CreateTablesIfNotExists` goes away.